contents/handbook/company/security.md

```diff
@@ -72,6 +72,10 @@ We receive data collected by our customers from end-users and allow them to unde
 
 We conduct these annually, most recently in April 2024  - <PrivateLink url="https://github.com/PostHog/company-internal/issues/1577">see here</PrivateLink> for a link to the latest report and statuses. 
 
+## Responsible disclosure
+
+Security vulnerabilities and other security related findings can be reported by emailing security@posthog.com. Valid findings will be rewarded with PostHog swag.
```

Comment:
I believe rewarding with swag is our current policy, but I couldn't find it documented. Can someone confirm?

Reply:
Yeah, confirmed, no cash rewards at the moment